superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -563,6 +563,7 @@ class Header extends React.PureComponent {
                               onClick={undoLength && onUndo}
                               data-test="undo-action"
                               iconSize="xl"
+                              role="button"
```

Comment:
Thanks @eulloa10 for all your works.

I propose moving the `onClick` event handler to the `StyledUndoRedoButton` component itself, rather than applying a button role to the icon within it.